core/src/test/java/com/linecorp/armeria/server/DefaultUnhandledExceptionReporterTest.java

```diff
@@ -124,4 +130,55 @@ void ignorableExceptionShouldNotBeReportedEvenThoughLoggingServiceIsPreviouslyDe
     }
+
+    @Test
+    void thrownExceptionsShouldBeIncrementedByTheNumberOfTimesTheSameExceptionOccurs() throws Exception {
+        Throwable cause = new IllegalArgumentException();
+        defaultUnhandledExceptionsReporter.report(ServiceRequestContext.of(createMockRequest()), cause);
+        defaultUnhandledExceptionsReporter.report(ServiceRequestContext.of(createMockRequest()), cause);
+        defaultUnhandledExceptionsReporter.report(ServiceRequestContext.of(createMockRequest()), cause);
+        assertThat(defaultUnhandledExceptionsReporter.getThrownExceptions().size()).isEqualTo(1);
+        assertThat(defaultUnhandledExceptionsReporter.getThrownExceptions().get(cause).first().longValue())
+                .isEqualTo(3);
+    }
+
+    @Test
+    void thrownExceptionsShouldBeAggregatedSeparatelyEachTimeNewExceptionOccurs() throws Exception {
+        Throwable cause1 = new IllegalArgumentException();
+        Throwable cause2 = new ArithmeticException();
+        Throwable cause3 = new NoSuchFieldException();
+        defaultUnhandledExceptionsReporter.report(ServiceRequestContext.of(createMockRequest()), cause1);
+        defaultUnhandledExceptionsReporter.report(ServiceRequestContext.of(createMockRequest()), cause2);
+        defaultUnhandledExceptionsReporter.report(ServiceRequestContext.of(createMockRequest()), cause3);
+        assertThat(defaultUnhandledExceptionsReporter.getThrownExceptions().size()).isEqualTo(3);
+        assertThat(defaultUnhandledExceptionsReporter.getThrownExceptions().get(cause1).first().longValue())
+                .isEqualTo(1);
+        assertThat(defaultUnhandledExceptionsReporter.getThrownExceptions().get(cause2).first().longValue())
+                .isEqualTo(1);
+        assertThat(defaultUnhandledExceptionsReporter.getThrownExceptions().get(cause3).first().longValue())
+                .isEqualTo(1);
+    }
+
+    @Test
+    void temp() throws Exception {
+        Throwable cause1 = new IllegalArgumentException();
+        Throwable cause2 = new IllegalArgumentException();
+        Throwable cause3 = new IllegalArgumentException();
+        defaultUnhandledExceptionsReporter.report(ServiceRequestContext.of(createMockRequest()), cause1);
+        defaultUnhandledExceptionsReporter.report(ServiceRequestContext.of(createMockRequest()), cause2);
+        defaultUnhandledExceptionsReporter.report(ServiceRequestContext.of(createMockRequest()), cause3);
+        assertThat(defaultUnhandledExceptionsReporter.getThrownExceptions().size()).isEqualTo(3);
+        assertThat(defaultUnhandledExceptionsReporter.getThrownExceptions().get(cause1).first().longValue())
+                .isEqualTo(1);
+        assertThat(defaultUnhandledExceptionsReporter.getThrownExceptions().get(cause2).first().longValue())
+                .isEqualTo(1);
+        assertThat(defaultUnhandledExceptionsReporter.getThrownExceptions().get(cause3).first().longValue())
+                .isEqualTo(1);
+    }
```

Comment:
@injae-kim nim, I have a question 🙇 
When referring to this test, even though it's the same `IllegalArgumentException`, it's being recognized as a new key because they are different objects. 
Is this the direction we want? I thought that even if they are different objects, they should be recognized as the same key if they have the same type. 😄 
Thank you~!

Reply:
Let's wait maintainer's comment here~! 🙇